pkg/storage/mysql/mysql.go

```diff
@@ -32,6 +32,7 @@ type MySQL struct {
 	stbl                   sq.StatementBuilderType
 	db                     *sql.DB
 	logger                 logger.Logger
+	dbStatsCollector       prometheus.Collector
```

Comment:
Why add the collector explicitly here instead of just using the default collector registration?

Reply:
We use the default, but I need to deregister after closing to have the tests work properly